invenio_files_rest/config.py

```diff
@@ -100,3 +107,9 @@
 
 FILES_REST_MULTIPART_EXPIRES = timedelta(days=4)
 """Time delta after which a multipart upload is considered expired."""
+
+FILES_REST_TASK_WAIT_INTERVAL = 2.0
```

Comment:
Why do we use `timedelta` for `FILES_REST_MULTIPART_EXPIRES` and seconds for `FILES_REST_TASK_WAIT_INTERVAL` and `FILES_REST_TASK_MAX_WAIT`? Can we unify the units or at least add the unit in docstring?